patterns/blueprint-vpc-lattice/cluster2/secrets.sh

```diff
@@ -0,0 +1,36 @@
+##!/bin/bash
+DBHOST="$(terraform output -raw postgres_host)"
```

Comment:
Got this error : 
```
│ Error: Unsupported value for raw output
│ 
│ The -raw option only supports strings, numbers, and boolean values, but
│ output value "postgres_host" is tuple.
│ 
│ Use the -json option for machine-readable representations of output values
│ that have complex types.
```

Try this : terraform output -json postgres_host | jq -r ".[0]"